app/assets/javascripts/classes/StatisticView.ts

```diff
@@ -22,14 +22,19 @@ interface StatisticView {
     id: string;
     name: string;
     spec: GraphSpec;
-    paused: boolean;
     targetFrameRate: number;
-    start(): void;
+    render(): void;
+    start(paused: boolean): void;
+    setPause(pause: boolean): void;
     postData(server: string, timestamp: number, dataValue: number, cvm: ConnectionVM): void;
     shutdown(): void;
     setViewDuration(duration: ViewDuration): void;
     updateColor(cvm: ConnectionVM): void;
     disconnectConnection(cvm: ConnectionVM): void;
+
+    renderDots: KnockoutObservable<boolean>;
```

Comment:
The corresponding values in the Series object don't generalize either.  There is definitely looks like there is work to be done to properly support multiple types of visualization.  For the scope of this PR I think having these values here follows the existing patterns.